static/js/components/ProfileProgressControls.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import Button from 'react-mdl/lib/Button';
+
+import { saveProfileStep } from '../util/profile_edit';
+
+export default class ProfileProgressControls extends React.Component {
+  static propTypes = {
+    nextUrl: React.PropTypes.string,
+    prevUrl: React.PropTypes.string,
+    isLastTab: React.PropTypes.bool,
+    saveProfile: React.PropTypes.func.isRequired,
```

Comment:
I agree that the use of `this.props` is kind of opaque and maybe we should send our arguments more explicitly. I don't think `profile.save()` would be clean or easy to implement however. To make changes to the state we need to dispatch actions, then we need to wait for React to send new props to have an update. It's not like a Django model where `save()` is synchronous and the model is updated right after the call.